src/commands/install-packages.yml

```diff
@@ -72,10 +72,11 @@ steps:
                 echo "Found yarn.lock file, assuming lockfile"
                 cp yarn.lock /tmp/node-project-lockfile
               fi
+              cp package.json /tmp/node-project-package.json
```

Comment:
I don't understand the purpose of copying the package.json. I thought the purpose of copying the lockfile was to collapse all the different cases down to one. It seems like it would be more efficient to use `ln` instead of `cp` throughout since these files are never modified, but again I don't see the need for this in the first place.

Reply:
I believe `ln` (hard link) rather than `ln -s` (symbolic link) is the droid you are looking for.

Reply:
Thanks :D